pygbm/loss.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+# TODO: ABC for losses, regression losses and classification losses
+class LeastSquares:
+    def __init__(self):
+        self.hessians = np.full(shape=1, fill_value=2, dtype=np.float32)
+
+    def get_gradients(self, y, y_pred):
+        return -2 * (y - y_pred).astype(np.float32)
```

Comment:
Cosmetics:

```python
return 2 * (y_pred - y).astype(np.float32)
```

Reply:
Maybe the `y` variable in the loss functions could be renamed to `y_true` (but not in the fit method where `y_train` / `y_val` is better.